.github/workflows/ci_ut_ext_blob_workflow.yml

```diff
@@ -0,0 +1,64 @@
+name: UT CI Run for Python Extension Blob
+
+on:
+    push:
+      branches: [ dev, master, main, release/* ]
+      paths:
+        - 'azure-functions-extension-base/**'
+        - 'azure-functions-extension-blob/**'
+    pull_request:
+        branches: [ dev, master, main, release/* ]
+        paths:
+          - 'azure-functions-extension-base/**'
+          - 'azure-functions-extension-blob/**'
+
+jobs:
+  build:
+    name: "Python Extension Blob UT CI Run"
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix:
+          python-version: [ 3.9, "3.10", "3.11" ]
+    permissions: read-all
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      working-directory: azure-functions-extension-blob
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -U -e .[dev]
+        python -m pip install -e ../azure-functions-extension-base
```

Comment:
We should test this with the published base. If there are base changes then we should publish the base then the extensions.

Reply:
will remove